Plots/ColormapComparisons/CB_Rain.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+CB_Rain.py
+================
+
+This script illustrates multiple color schemes for color maps which will allow for those
+impacted by color blindness to see visualizations. Using rainbow color schemes not only
+cause confusion for those impacted by color blindness, but is a poor choice in color scheme 
+for images that may be transferred to a black and white scale for printing.
+This code addresses a handful of options to use in place of rainbow color schemes for 
+use in the matplotlib.pyplot library.
+
+More information on this subject can be found here:
+        -https://agilescientific.com/blog/2017/12/14/no-more-rainbows
+        -https://www.researchgate.net/publication/328361220_The_Effect_of_Color_Scales_on_Climate_Scientists'_Objective_and_Subjective_Performance_in_Spatial_Data_Analysis_Tasks
```

Comment:
Wonderful idea including the links! This can help further educate scientists on proper color maps.

Reply:
Also, this made me wonder if we should add a page to the gallery specifically mentioning the various resources we have looked at as a group about color maps. @clyne @kmpaul What do you two think?